herddb-core/src/main/java/herddb/storage/DataStorageManager.java

```diff
@@ -64,6 +64,10 @@
     public abstract List<Record> readPage(String tableSpace, String uuid, Long pageId)
         throws DataStorageManagerException, DataPageDoesNotExistException;
 
+    public abstract void ensureIndexDirectory(String tableSpace, String indexName) throws DataStorageManagerException;
+    
```

Comment:
I cannot find invocation for BRIN indexes (they too use writeIndexPage if I remember well)

Reply:
good catch ! added